### PR TITLE
Review topics of interest

### PR DESCRIPTION
**Pull request by fermadeiral**

### CODE REVIEW

File: cfp.html [hunk 1]
```diff
@@ -100,7 +100,7 @@ <h3>Call For Papers</h3>
                         <h2>Topics of Interest</h2>
                         <div class="description-one">
                             <p align="justify">
-                                VEM provides a forum where researchers and practitioners from academia and industry can present studies, tools, and data related to topics on software visualization, maintenance, and evolution.
+                                VEM provides a forum where researchers and practitioners from academia and industry can present and discuss research related to topics on software visualization, maintenance, and evolution.
                                 Topics of interest include, but are not limited to:
```
Comment (felipeebert) [inline cfp.html:104]: I guess we could add a line break before the list, what do you think?

File: cfp.html [hunk 2]
```diff
@@ -128,6 +128,10 @@ <h2>Topics of Interest</h2>
                                 <li>Visualization of or to support software development activities (e.g., software testing)</li>
                                 <li>Visualization-based techniques in software engineering education</li>
                             </ul>
+                            <p>
+                                Papers presenting studies, tools, datasets, or innovative ideas are welcome.
+                                Experimental evaluation of new tools and ideas is desirable, but it is not a precondition to submit a paper to the workshop.
```
Comment (felipeebert) [inline cfp.html:133]: The last sentence isn't related to the submission/evaluation instead of topics of interest?

**Review by felipeebert — COMMENTED** [review]
Minor suggestions.

File: cfp.html (hunk 2 above)
Comment (fermadeiral) [reply cfp.html:133]: I'm not sure. To me, this is more about "scope", and the place we are talking about scope is the topics of interest.

Comment (felipeebert) [reply cfp.html:133]: Ah, ok, this is not the the list of topics... sorry. that's fine then. =D

Comment (fermadeiral) [reply cfp.html:133]: Felipe, it is in the topics. Not in the list, but in the same section.

Comment (felipeebert) [reply cfp.html:133]: hm, the topics shouldn´t be under the <ul>
but I still think it´s weird to be listed as a topic...

Comment (fermadeiral) [reply cfp.html:133]: I don't understand your comment.

Comment (felipeebert) [reply cfp.html:133]: Please ignore my comments. I was misunderstanding it. It's all right the way it is.

Comment (fermadeiral) [reply cfp.html:133]: Ok!

File: cfp.html (hunk 1 above)
Comment (fermadeiral) [reply cfp.html:104]: I have added a spacing there instead of a line break.

**Comment by fermadeiral** [comment]
Ready for review again.

**Pull request merged**